### ISSUE TITLE
Fix seedling deployment range tooltips

### ISSUE THREAD

**Issue by daviswx** [ncsco/pinemap-dss-help#74]
The tooltips in the seedling deployment range legends (below the maps) are currently turned off.

The JQuery code that handles the tooltip behavior doesn't support tooltips like these that are added after the page has loaded.  I can modify this code to handle those, but then it introduces another problem: When the tooltips in the Projected Average map titles are clicked, it enlarges the maps as well as displaying the tooltips.

I can create a separate function to handle just the risk range tooltips, but I didn't have time to do this before the public rollout.

**Comment by daviswx** [comment]
One other idea is adding a small asterisk next to the marker, like this:
![seed_deployment_risk_range_legend_asterisk](https://cloud.githubusercontent.com/assets/9257580/11593362/d97f0108-9a71-11e5-9c68-43c5bff7a2b3.PNG)

That would avoid the need for a tooltip icon, which can also clutter up the space below that legend a bit.  But would it be obvious enough that people need to interact with it in some way (i.e., hover their mouse over it) to see an explanation?

**Comment by hadinon** [comment]
I really like this asterisk idea. Instead of getting people to hover over it to see the explanation, would it be feasible to put the explanation right below like within each legend container?

**Comment by daviswx** [comment]
Here's how it would look to have the explanations beneath the legends:

**Seedling Markets tool**
![seedling_markets_legend_explanation](https://cloud.githubusercontent.com/assets/9257580/11755122/2bb4bd1c-a01e-11e5-8a27-5206e81ac474.PNG)

**Seedling Sources tool**
![seedling_sources_legend_explanation](https://cloud.githubusercontent.com/assets/9257580/11755123/2e79de10-a01e-11e5-8ece-ca6094c68506.PNG)

**Comment by hadinon** [comment]
Hmmm it does seem a bit cluttered but maybe not too bad. I would be OK with this. What are your thoughts?

**Comment by hadinon** [comment]
One minor thing that I just noticed: the seedling sources text says, "...minimum/mean/maximum projected temperature" instead of "lowest likely / multi-model mean / highest likely". I'm sure this is on your radar to fix but just wanted to make sure!

**Comment by daviswx** [comment]
Of all the options we've tried so far, I think I like this one the best.  My only concern is that it may be a bit wordy, especially since it wraps to a third line on the Seedling Sources tool.  I have updated the wording for that tool as suggested.

**Comment by hadinon** [comment]
Yeah, I agree. With the updated wording, it is no longer wrapping to a third line on the Seedling Sources tool so I'm going to close this. Please re-open if needed.
